### ISSUE TITLE
--rspec option not respected on gem creation

### ISSUE THREAD

**Issue by dgmora** [technicalpickles/jeweler#277]
With the following options (in 2.0.1) :

    jeweler --rspec --shoulda  --user-name dgmora --user-email david@somewhere.com --github-username dgmora --git-remote heythere --create-repo --yard heythere

rspec folder is not created, TestUnit is used instead:

	create	.gitignore
	create	Rakefile
	create	Gemfile
	create	LICENSE.txt
	create	README.rdoc
	create	.document
	create	lib
	create	lib/heythere.rb
	create	test
	create	test/helper.rb
	create	test/test_heythere.rb

If I **swap** `--rspec` and `--shoulda` options, the rspec folder is generated as expected.

**Comment by flajann2** [comment]
Is this still the case? If so I will fix it shortly.

**Comment by dgmora** [comment]
yes, it is. I am not sure though, that those flags where expected to be used together (or what should it produce). I try that because sometimes with Rails I used RSpec + Shoulda together

**Comment by technicalpickles** [comment]
Jeweler's use of shoulda predates when it could be used with RSped as matchers. I'm not kept up with shoulda to know if it is even usable by itself without RSpec. If it can't then it'd make sense to remove/deprecate the option.
